lobsterpy/cli.py

```diff
@@ -36,108 +37,159 @@ def get_parser() -> argparse.ArgumentParser:
-        # nargs="+",
-        help="List of potcar symbols",
-    )
 
-    input_file_group.add_argument(
-        "--lobsterin",
-        default="lobsterin",
+    icohplist_file = argparse.ArgumentParser(add_help=False)
+    icohplist_file.add_argument(
+        "-ficohp",
+        "--file-icohplist",
+        default="ICOHPLIST.lobster",
+        dest="icohplist",
         type=Path,
-        help='path to lobsterin. Default is "lobsterin".',
+        help='path to ICOHPLIST.lobster. Default is "ICOHPLIST.lobster"',
     )
 
-    input_file_group.add_argument(
-        "--lobsterout",
-        default="lobsterout",
+    coxxcar_file = argparse.ArgumentParser(add_help=False)
+    coxxcar_file.add_argument(
+        "-fcohp",
+        "--file-cohpcar",
+        default="COHPCAR.lobster",
+        dest="cohpcar",
         type=Path,
-        help='path to lobsterout. Default is "lobsterout".',
+        help='path to COHPCAR.lobster. Default is "COHPCAR.lobster". This argument'
+        "can also read COBICARs or COOPCARs. One needs to use appropriate --cobis or "
+        "--coops options along with this argument when plotting",
     )
 
-    input_file_group.add_argument(
-        "--doscar",
+    doscar_file = argparse.ArgumentParser(add_help=False)
+    doscar_file.add_argument(
+        "-fdos",
+        "--file-doscar",
         default="DOSCAR.lobster",
+        dest="doscar",
         type=Path,
         help='path to DOSCAR.lobster. Default is "DOSCAR.lobster".',
     )
 
-    input_file_group.add_argument(
-        "--vasprun",
+    user_basis_arg = argparse.ArgumentParser(add_help=False)
+    user_basis_arg.add_argument(
+        "--userbasis",
+        "--user-basis",
+        default=None,
+        type=_element_basis,
+        nargs="+",
+        help="Use the specific basis provided by the user to "
+        "generate the inputs (e.g.,  --userbasis Cr.3d.3p.4s N.2s.2p). "
+        "Default is None.",
+    )
+
+    # groups of arguments for specific actions
+
+    calc_quality_description_file_parent = argparse.ArgumentParser(add_help=False)
+
+    # Input args for specifically needed for calculation quality description
+    calc_quality_description_file_group = (
+        calc_quality_description_file_parent.add_argument_group()
+    )
+    calc_quality_description_file_group.add_argument(
+        "-fvasprun",
+        "--file-vasprun",
         default="vasprun.xml",
+        dest="vasprun",
         type=Path,
         help='path to vasprun.xml. Default is "vasprun.xml".',
     )
-
-    input_file_group.add_argument(
-        "--bandoverlaps",
+    calc_quality_description_file_group.add_argument(
+        "-fbandoverlap",
+        "--file-bandoverlaps",
         default="bandOverlaps.lobster",
+        dest="bandoverlaps",
         type=Path,
         help='path to bandOverlaps.lobster. Default is "bandOverlaps.lobster".',
     )
+    calc_quality_description_file_group.add_argument(
+        "-flobsin",
```

Comment:
I think I would rather call it -flobsterin

Reply:
Ah yeah did not think this way, will change it to `-flobsterin`